### PR TITLE
Implemented TOUCH command

### PR DESCRIPTION
**Pull request by sazzad16**

### CODE REVIEW

File: src/test/java/redis/clients/jedis/tests/commands/AllKindOfValuesCommandsTest.java [hunk 1]
```diff
@@ -356,6 +356,46 @@ public void ttl() {
   }
 
   @Test
+  public void touch() {
```
Comment (marcosnils) [inline src/test/java/redis/clients/jedis/tests/commands/AllKindOfValuesCommandsTest.java]: What about adding an `OBJECT IDLETIME <key>` call in some of the examples to validate that Jedis is actually running the `touch` command?. This way we can verify that the access time of the specified key(s) was actually changed.

**Comment by HeartSaVioR** [comment]
LGTM. @marcosnils

**Comment by sazzad16** [comment]
#1412

**Comment by marcosnils** [comment]
Merged to master and 2.10 respectively. Thx @sazzad16 for your contributions!.

**Pull request merged**